src/components/config/validate.ts

```diff
@@ -42,6 +42,19 @@ import { RequestConfig } from '../../interfaces/request'
 import { Validation } from '../../interfaces/validation'
 import { isValidURL } from '../../utils/is-valid-url'
 
+const HTTPMethods = [
+  'DELETE',
+  'GET',
+  'HEAD',
+  'OPTIONS',
+  'PATCH',
+  'POST',
+  'PUT',
+  'PURGE',
+  'LINK',
+  'UNLINK',
```

Comment:
Personally, I've never heard purge, link, and unlink method. Most of browser didn't use http method that is not officially in RFC. [This is http methods](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods) used by mozilla (firefox).
Do we need to have those purge, link, and unlink methods?
Or can we just use [restful http methods](https://restfulapi.net/http-methods/) + options (preflight)?

Reply:
I'm basically with you with the purge, link, unlink method. But when I import the Method type from axios, it is defined like this:

![image](https://user-images.githubusercontent.com/16670475/127274821-de267685-3ca0-479f-890b-03aeddfc8901.png)

I just matched the array with the type